app/Models/Group/Workspace.php

```diff
@@ -34,6 +35,11 @@ public function group()
         return $this->belongsTo(Group::class);
     }
 
+    public function topicSections()
```

Comment:
If there isn't or will not be a "Workspace Section" model, I would simply call this `sections()`. 

if there is or will be a "Workspace Section", then this clarification makes sense.

Reply:
There could be. I don't know yet, so just keeping it clear.